### ISSUE TITLE
Migrate code into dedicated repo(s) for Substrate module 

### ISSUE THREAD

**Issue by nuke-web3** [Polkadot-Blockchain-Academy/pba-content#718]
This repo should include only student-facing slides and instructor-facing notes on how to run the show :teacher: 

Looks like these are the only code examples we should move out into either a monorepo for the module, or if intended to be packaged and shared independently in separate repo(s):

- [x] syllabus/4-Substrate/frameless-node-template
   - https://github.com/Polkadot-Blockchain-Academy/PBA2-BA-frameless-node-template and or https://github.com/Polkadot-Blockchain-Academy/UNGRADED-frameless-node-template-PBA-BA-2023-to-delete are more updated IIRC than what exists in the content repo here.
- [x] syllabus/4-Substrate/wasm-crate
- [x] syllabus/4-Substrate/wasm-crate-bin
- [x] #857

**Comment by wirednkod** [comment]
I do not think this neds to be still open. @kianenigma can you elaborate please?
